dev-ml/ocaml-cstruct/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <herd>ml</herd>
+    <herd>proxy-maintainers</herd>
+    <maintainer>
+        <email>tomboy64@sina.cn</email>
+    </maintainer>
+        <use>
```

Comment:
hmm. i figured i fixed this.
gonna check that out. multiple branches are really annoying with this kind of mass-replace.